src/guidance/translator.cpp

```diff
@@ -0,0 +1,150 @@
+#include <cstddef>
```

Comment:
OSRM output should be symbolic and language agnostic-only. It is okay to expose the symbolic modifiers, but we should not include textual information here.

Problems I'm seeing with that would be:
- Translations
- HTML wrapped text
- Typo fix would mean we need to roll a new OSRM release

This code should be easy to translate for `api-directions`/`osrm-frontend` though.